packngo.go

```diff
@@ -230,6 +231,45 @@ func (c *Client) Do(req *http.Request, v interface{}) (*Response, error) {
 	return &response, err
 }
 
+// dumpDeprecation logs headers defined by
+// https://tools.ietf.org/html/rfc8594
+func dumpDeprecation(resp *http.Response) {
+	deprecation := resp.Header.Get("Deprecation")
```

Comment:
This value may be either "true" or a date, I haven't handled the "true" value.
https://tools.ietf.org/html/draft-ietf-httpapi-deprecation-header-01#section-2.1